src/backend/distributed/operations/citus_tools.c

```diff
@@ -53,13 +56,44 @@ static void ExecuteCommandsAndStoreResults(StringInfo *nodeNameArray,
 										   StringInfo *resultStringArray,
 										   int commandCount);
 static bool ExecuteRemoteQueryOrCommand(char *nodeName, uint32 nodePort,
-										char *queryString, StringInfo queryResult);
+										char *queryString, StringInfo queryResultString);
```

Comment:
This change was not intended.

```suggestion
										char *queryString, StringInfo queryResult);
```